### PR TITLE
Brick color scheme in calibration plots and other small changes

### PR DESCRIPTION
**Pull request by ricardarosemann**
- Adopt the Brick color schemes in calibration plots and additionally include specific color for "no change" renovation flows, both via ```mip::plotstyle()```.
- Flexibilize the framework to generate calibration plots: Now allows to pass facets and extending the y-axis to zero can now be inactivated.
- Correct output file name and one error message when generating the plots.

### CODE REVIEW

**Review by robinhasse — CHANGES_REQUESTED** [review]
Looks good. Just wodering about the local path...

File: inst/plotsCalibrationReporting/plotsCalibration.Rmd [hunk 1]
```diff
@@ -9,7 +9,7 @@ output:
 params:
     docTitle: "BRICK calibration report"
     cal: "BRICK_calibration_report.csv"
-    path: "C:\\Users\\ricardar\\Documents\\Results\\BRICK\\calibration-tests\\extrElec-vin\\lowEhp1-2010-30Iter0_1A"
+    path: "C:\\Users\\ricardar\\Documents\\Results\\BRICK\\calibration-tests\\extrElec-vin"
```
[the remark this answers is not in the document]
Comment (ricardarosemann) [reply inst/plotsCalibrationReporting/plotsCalibration.Rmd:12]: Thank you for pointing to this. This is usually overwritten by ```plotBRICKCalib()``` and only applies if the ```.Rmd``` file is executed directly. But still doesn't look very nice. I'll pass the current working directory instead.

**Comment by ricardarosemann** [comment]
I also fixed a naming inconsistency (function ```plotBRICKCalib()``` vs. file name ```plotBRICKcalib.R```), restricted the ```dplyr``` import to the required functions and adjusted further defaults in the header of the ```.Rmd``` file.

I noticed that knitting the ```.Rmd``` file directly does not work properly, because it cannot find ```readBrickSets()``` (I guess because it is not exported). But I figure this is a very rare use case, so I don't think this will be a problem.

**Review by robinhasse — APPROVED** [review]
Thanks you!

**Pull request merged**